Version 2.0.0-beta.14 fails type check for typecsript

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#reporting-new-issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [x] I have tried the `npm run clear` or `yarn clear` command.
- [x] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [ ] I have tried creating a repro with https://new.docusaurus.io.
- [x] I have read the console error message carefully (if applicable).

### Description

Latest version of docusaurus fails typescript tsc with error `Cannot find module '@theme/BlogSidebar' or its corresponding type declarations.`

### Steps to reproduce

Init a new folder, and add packages and tsconfig as outlined here https://docusaurus.io/docs/typescript-support#setup

Then add a new command to your package.json `"typecheck": "tsc --noEmit"`

Finally run yarn typecheck

### Expected behavior

Fresh install has no errors.

### Actual behavior

There are fiver errors. `Cannot find module '@theme/BlogSidebar' or its corresponding type declarations.`

### Your environment

- Public source code:
- Public site URL:
- Docusaurus version used: 2.0.0-beta.14
- Environment name and version (e.g. Chrome 89, Node.js 16.4): Chrome Version 96.0.4664.110, Node v16.3.0
- Operating system and version (e.g. Ubuntu 20.04.2 LTS): Mac Big sur 11.6.1


### Reproducible demo

Could not get terminal to work in code sandbox but this is very minimal to setup

### Self-service

- [ ] I'd be willing to fix this bug myself.

--- a/website/docs/typescript-support.md
+++ b/website/docs/typescript-support.md
@@ -27,8 +27,7 @@ Then add `tsconfig.json` to your project root with the following content:
 
 ```json title="tsconfig.json"
 {
-  "extends": "@tsconfig/docusaurus/tsconfig.json",
-  "include": ["src/"]
+  "extends": "@tsconfig/docusaurus/tsconfig.json"
 }
 ```
 

